File: java/03_if.java
public class IfExample {
    public static void main(String[] args) {
        int age = 20;

        if (age >= 18) {
            System.out.println("adult");
        } else {
            System.out.println("not adult yet");
        }

        int score = 85;

        if (score >= 90) {
            System.out.println("A");
        } else if (score >= 80) {
            System.out.println("B");
        } else if (score >= 70) {
            System.out.println("C");
        } else {
            System.out.println("need more work");
        }
    }
}

File: php/03_if.php
<?php
$age = 20;

if ($age >= 18) {
    echo "adult\n";
} else {
    echo "not adult yet\n";
}

$score = 85;

if ($score >= 90) {
    echo "A\n";
} elseif ($score >= 80) {
    echo "B\n";
} elseif ($score >= 70) {
    echo "C\n";
} else {
    echo "need more work\n";
}
